File: baseble/src/main/java/com/fcbox/ble/baseble/callback/IRssiCallback.java
package com.fcbox.ble.baseble.callback;


import com.fcbox.ble.baseble.exception.BleException;


/**
 * 获取信号值回调
 */
public interface IRssiCallback {
    void onSuccess(int rssi);

    void onFailure(BleException exception);
}

File: baseble/src/main/java/com/xzy/ble/baseble/callback/IRssiCallback.kt
package com.xzy.ble.baseble.callback


import com.xzy.ble.baseble.exception.BleException


/**
 * 获取信号值回调
 */
interface IRssiCallback {
    fun onSuccess(rssi: Int)

    fun onFailure(exception: BleException)
}
